### ISSUE TITLE
Megamenu con più voci

### ISSUE THREAD

**Issue by AlessandroVecchi** [italia/bootstrap-italia#654]
Ciao,
Una parte critica del megamenu è il modo in cui si naviga da tastiera tra le voci principali e le voci secondarie.
Attualmente l'esempio del componente ha una sola voce principale pertanto non è possibile testare la correttezza dello spostamento tra le voci.

## Comportamento atteso

Tramite il tab ci si dovrebbe spostare tra le voci dello stesso livello senza entrare in quelle secondarie.

## Comportamento attuale

Non verificabile.

Ho notato altresì che il menu dropdown si apre con la freccia in giù ma non si chiude con la freccia in su. Come logica secondo me dovrebbe farlo però non so esattamente quel è il comportamento che dovrebbe avere per l’accessibilità.
Lo spostamento nelle sottovoci avviene solo con cursore su/giù e non dx/sx. Inoltre dopo l’ultima voce non è possibile ritornare alla prima se non scorrendole a ritroso. Sono comportamenti voluti?

## Possibili soluzioni

Aggiungere più voci di menu principali.
Verificare gli altri punti segnalati.

**Comment by astagi** [comment]
Grazie del report @AlessandroVecchi 

1. La parte di navigazione con tastiera è stata verificata e concordata con AGID. 

2. Per la parte menu dropdown l'indicatore mi funziona correttamente, che browser stai usando?

![image](https://user-images.githubusercontent.com/537363/160583225-21fe3c92-9a54-4f9d-8886-d125cf8f3205.png)
![image](https://user-images.githubusercontent.com/537363/160583284-4631b855-2c06-44f3-94a2-31080676a952.png)

3. Fornisco qua un esempio di doppio megamenu per non creare entropia nella documentazione

![image](https://user-images.githubusercontent.com/537363/160583046-b8e8341d-80f6-40ea-b066-27153cafa2be.png)

```html
<nav class="navbar navbar-expand-lg has-megamenu">
  <button class="custom-navbar-toggler" type="button" aria-controls="nav1" aria-expanded="false" aria-label="Mostra/Nascondi la navigazione" data-target="#nav1">
    <svg class="icon">
      <use href="{{site.baseurl}}/dist/svg/sprite.svg#it-burger"></use>
    </svg>
  </button>
  <div class="navbar-collapsable" id="nav1">
    <div class="overlay"></div>
    <div class="close-div">
      <button class="btn close-menu" type="button">
        <span class="sr-only">Nascondi la navigazione</span>
        <svg class="icon">
          <use href="{{site.baseurl}}/dist/svg/sprite.svg#it-close-big"></use>
        </svg>
      </button>
    </div>
    <div class="menu-wrapper">
      <ul class="navbar-nav">
        <li class="nav-item dropdown megamenu">
          <a class="nav-link dropdown-toggle" href="#" data-toggle="dropdown" aria-expanded="false" id="megamenu1">
            <span>Megamenu 1</span>
            <svg class="icon icon-xs">
              <use href="{{site.baseurl}}/dist/svg/sprite.svg#it-expand"></use>
            </svg>
          </a>
          <div class="dropdown-menu" role="region" aria-labelledby="megamenu1">
            <div class="row">
              <div class="col-12 col-lg-4">
                <div class="link-list-wrapper">
                  <ul class="link-list">
                    <li><a class="list-item" href="#"><span>Link lista 1</span></a>
                    </li>
                    <li><a class="list-item" href="#"><span>Link lista 2</span></a>
                    </li>
                    <li><a class="list-item" href="#"><span>Link lista 3</span></a>
                    </li>
                  </ul>
                </div>
              </div>
              <div class="col-12 col-lg-4">
                <div class="link-list-wrapper">
                  <ul class="link-list">
                    <li><a class="list-item" href="#"><span>Link lista 4</span></a>
                    </li>
                    <li><a class="list-item" href="#"><span>Link lista 5</span></a>
                    </li>
                    <li><a class="list-item" href="#"><span>Link lista 6</span></a>
                    </li>
                  </ul>
                </div>
              </div>
              <div class="col-12 col-lg-4">
                <div class="link-list-wrapper">
                  <ul class="link-list">
                    <li><a class="list-item" href="#"><span>Link lista 7</span></a>
                    </li>
                    <li><a class="list-item" href="#"><span>Link lista 8</span></a>
                    </li>
                    <li><a class="list-item" href="#"><span>Link lista 9</span></a>
                    </li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li class="nav-item dropdown megamenu">
          <a class="nav-link dropdown-toggle" href="#" data-toggle="dropdown" aria-expanded="false" id="megamenu2">
            <span>Megamenu 2</span>
            <svg class="icon icon-xs">
              <use href="{{site.baseurl}}/dist/svg/sprite.svg#it-expand"></use>
            </svg>
          </a>
          <div class="dropdown-menu" role="region" aria-labelledby="megamenu2">
            <div class="row">
              <div class="col-12 col-lg-12">
                <div class="link-list-wrapper">
                  <ul class="link-list">
                    <li><a class="list-item" href="#"><span>Link lista 1</span></a>
                    </li>
                    <li><a class="list-item" href="#"><span>Link lista 2</span></a>
                    </li>
                    <li><a class="list-item" href="#"><span>Link lista 3</span></a>
                    </li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
        </li>
      </ul>
    </div>
  </div>
</nav>
```

**Comment by AlessandroVecchi** [comment]
Ciao,
Testato con Firefox e Chrome aggiornati su win10.

Notavo adesso che il dropdown si apre indifferentemente col cursore su e giù ma si chiude solo con ESC.

**Comment by astagi** [comment]
@AlessandroVecchi si cursori oppure invio e chiusura con ESC. In ogni caso la freccia almeno per i test che ho effettuato io funziona correttamente, se puoi allegare una registrazione dello schermo per mostrare cosa succede vediamo di capire come risolvere. Grazie :)

**Comment by AlessandroVecchi** [comment]
Ho fatto il video però da questo non si evincono le azioni che sto compiendo
https://drive.google.com/file/d/1gi0utLDOvp5ky4N89xtNJWDP4MefKUau/view?usp=sharing

**Comment by astagi** [comment]
@AlessandroVecchi perdonami ma lo vedo piuttosto sgranato, non vedo la freccia come si comporta.. è possibile avere una qualità migliore?

**Comment by AlessandroVecchi** [comment]
> @AlessandroVecchi perdonami ma lo vedo piuttosto sgranato, non vedo la freccia come si comporta.. è possibile avere una qualità migliore?

prova adesso

**Comment by astagi** [comment]
Grazie mille @AlessandroVecchi si il comportamento mi sembra corretto, al click esterno al menu o alla pressione di ESC la freccia torna come prima giusto? la navigazione avviene tramite su,giù,tab e l'apertura tramite ENTER o su/giù

**Comment by AlessandroVecchi** [comment]
Esatto.
Questo è il comportamento corretto che deve avere per chi naviga da tastiera o con altri ausili alla navigazione?
è la tua risposta definitiva? la accendiamo? :)

**Comment by astagi** [comment]
@AlessandroVecchi Ok si avevo capito ci fossero problemi sull'indicatore :) si questo è il comportamento concordato e revisionato con AGID come dicevo in una precedente risposta, quindi si accendiamo :) Se non ci sono altri punti chiuderei la issue nel caso riapriamo per questioni più specifiche, ok?
